pkg/utils/testutils.go

```diff
@@ -2,12 +2,29 @@ package utils
 
 import (
 	"context"
+	"encoding/json"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
+
 	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
 )
 
 // Deprecated: use tests.Context
 func Context(t *testing.T) context.Context {
 	return tests.Context(t)
 }
+
+// A custom transformer for comparing JSON strings.
+//
+// Use this with the cmp.Diff function to compare JSON strings.
+//
+// E.g. cmp.Diff(expected, actual, transformJSON)
+var TransformJSON = cmp.FilterValues(func(x, y []byte) bool {
```

Comment:
PR #473 should be refactored after this is merged to use this rather than duplicate the same func

Reply:
Nit: if this only exists to be used with cmp.Diff, then maybe we could create an assertEqualsWithDiff that combines the two?

Reply:
Good idea, I can make a CmpJSON wrapper!